src/components/notes/EditorCorner.jsx

```diff
@@ -3,18 +3,20 @@ import PropTypes from 'prop-types'
 
 import Icon from 'cozy-ui/react/Icon'
 import Tooltip from 'cozy-ui/react/Tooltip'
+import { useI18n } from 'cozy-ui/react/I18n'
 
 import SharingWidget from 'components/notes/sharing'
 
 const EditorCorner = ({ doc, isPublic, isReadOnly }) => {
   if (!isPublic) return <SharingWidget file={doc.file} />
-  else if (isReadOnly)
+  else if (isReadOnly) {
+    const { t } = useI18n()
     return (
-      <Tooltip title={'This note is in read-only mode.'}>
+      <Tooltip title={t('Notes.Editor.read_only')}>
         <Icon icon="lock" color="var(--primaryTextColor)" />
       </Tooltip>
     )
-  else return false
+  } else return false
```

Comment:
Don"t we return null usually?

Reply:
Man I can never remember if it's null or false. But you're right it's null.